metro.config.js

```diff
@@ -25,6 +25,15 @@ const config = {
         // When we run the e2e tests we want files that have the extension e2e.js to be resolved as source files
         sourceExts: [...(isE2ETesting ? e2eSourceExts : []), ...defaultSourceExts, 'jsx'],
     },
+    // We are merging the default config from Expo and React Native and expo one is overriding the React Native one so inlineRequires is set to false so we want to set it to true
+    // for fix cycling dependencies and improve performance of app startup
+    transformer: {
+        getTransformOptions: async () => ({
+            transform: {
+                inlineRequires: true,
```

Comment:
Enabling it caused [this](https://github.com/Expensify/App/blob/48520ecc0f6389f72c8c136984893c4a99f79fc0/src/pages/workspace/taxes/WorkspaceTaxesPage.tsx#L31) top level code to be not immediately evaluated when we named import it. This caused https://github.com/Expensify/App/issues/56158#issuecomment-2650228394 and https://github.com/Expensify/App/issues/58322#issuecomment-2774869670